bridge_ui/src/utils/consts.ts

```diff
@@ -291,6 +291,13 @@ export const ALEPHIUM_HOST =
     ? "http://localhost:12973"
     : "http://localhost:22973"
 
+export const ALEPHIUM_EXPLORER_HOST =
+  CLUSTER === "mainnet"
+    ? "http://localhost:9090"
+    : CLUSTER === "testnet"
+    ? "http://localhost:9090"
+    : "http://localhost:9090"
```

Comment:
We could use the same configuration as here: https://github.com/alephium/desktop-wallet/blob/5f10be35b12cef027c47876230652c57a5453a73/src/utils/settings.ts

Reply:
It's for consistent with the legacy code, all configs use this style in the bridge UI code, and there are lots of codes like `CLUSTER === "mainnet"`.

Reply:
Alright then!